quixstreams/models/topics/topic.py

```diff
@@ -144,18 +145,24 @@ def row_serialize(self, row: Row, key: Any) -> KafkaMessage:
             raise SerializerIsNotProvidedError(
                 f'Value serializer is not provided for topic "{self.name}"'
             )
+
+        ctx = partial(SerializationContext, topic=self.name, headers=row.headers)
```

Comment:
Hey, as a small optimization, I suggest not using `partial` and initializing the `SerializationContext` directly here.

It's more verbose, but deserialization happens for each message, and partials do additional attribute lookups internally and are slightly slower than a normal function call.

Reply:
Fixed!